src/main/java/seedu/address/logic/command/copy/MoveCommand.java

```diff
@@ -61,7 +61,11 @@ public CommandResult execute(Model model) throws CommandException {
             model.copyActivity(activityToMove, travelPlanIndex);
             model.deleteActivity(activityToMove);
 
-            return new CommandResult(String.format(MESSAGE_MOVE_ACTIVITY_SUCCESS, activityToMove, travelPlan));
+            assert travelPlan.getActivityList().hasActivity(activityToMove) : "Activity was not moved";
```

Comment:
can replace this with:

`assert model.hasTravelPlanObject(activityToMove, travelPlanIndex.getZeroBased()) : "Activity was not moved";`

This is a new method I recently added to model. Will merge my PR soon. Can pull from upstream to retrieve.